packages/contact-exchange/src/services/OOBParser.ts

```diff
@@ -0,0 +1,43 @@
+import { OutOfBandInvitation } from './DIDCommOOBInvitation';
+
+export interface ParsedOOB {
+  invitation: OutOfBandInvitation;
+  encodedPart: string;
+}
+
+export function parseOOBInvitation(url: string): ParsedOOB | null {
+  const urlPattern = /^(https:\/\/mediator\.rootsid\.cloud\?_oob=)(.*)$/;
```

Comment:
this validation is not correct. 

![image](https://github.com/user-attachments/assets/2c9f5be7-cda3-4f48-b3fe-03336f895357)

The domain is not constrained to that url.

Reply:
Can you please provide me this link @hugoib ?

Reply:
https://identity.foundation/didcomm-messaging/spec/#standard-message-encoding